src/views/NavBar.jsx

```diff
@@ -0,0 +1,250 @@
+import * as React from 'react';
+import { SignInButton, SignOutButton, useAuth } from '../api/useAuth.jsx';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import { NavLink } from 'react-router-dom';
+import SvgIcon from '@mui/material/SvgIcon';
+
+function NavBar() {
```

Comment:
this is really more of a component than it is a view so I think this should live in ../components